test-browser/tests/ballot.js

```diff
@@ -1,35 +1,27 @@
 'use strict'
+var contractHelper = require('../helpers/contracts')
+var examples = require('../../src/app/example-contracts')
 
-var testData = require('../mockcompiler/requests')
-// var contractHelper = require('../helpers/contracts')
+var sources = {
+  'sources': {
+    'Untitled': examples.ballot.content
+  }
+}
 
 module.exports = {
+  '@Sources': function () {
+    return sources
+  },
   'Ballot': function (browser) {
-    runTests(browser, testData)
+    runTests(browser)
   }
 }
 
 function runTests (browser, testData) {
   browser
     .url('http://127.0.0.1:8080/#version=builtin')
     .waitForElementVisible('.newFile', 10000)
-  browser.assert.notEqual(testData, null)
-// TODO add Ballot tests. -> setValue('#input textarea', ... ) is not working properly with that contract.
-/*
-  testBallot(browser, testData.ballot.sources.Untitled, function () {
+  contractHelper.testContracts(browser, sources.sources.Untitled.replace(/(\n)/g, ' '), ['Ballot'], function () {
```

Comment:
i will move it to 
https://github.com/ethereum/browser-solidity/pull/283/files#diff-f7a6123b8a31b64951b350d64289b3abR24